notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -202,7 +202,6 @@
       "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #148.    # Sampling distributions for categorical features...
</pre><p>These lists should be in separate cell with an explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/841/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>fixed</p>